kubemarine/resources/configurations/globals.yaml

```diff
@@ -508,12 +514,18 @@ compatibility_map:
         sha1: 326110dcb62b66e69490d039b170682fb71c5560
       v1.23.11:
         sha1: 07769c846e4a83d59f9f34370c33be5cc163120b
+      v1.23.17:
+        sha1: ce74875b3802f4a9ac5dbd32a3f4c684b9ee4fd3
       v1.24.0:
         sha1: ce74875b3802f4a9ac5dbd32a3f4c684b9ee4fd3
       v1.24.2:
         sha1: 35c3d20f92c8159b4f65aaafe6e9fc57c9f9e308
+      v1.24.11:
```

Comment:
need to update _defaults.yaml_ **kubernetesVersion** with new default 1.24 version